lib/driver.js

```diff
@@ -18,6 +18,7 @@ import { setLocale, setPreferences } from './settings';
 import { runSimulatorReset, isolateSimulatorDevice, checkSimulatorAvailable,
          moveBuiltInApp, getAdjustedDeviceName, endSimulator, runRealDeviceReset } from './device';
 import { version } from '../../package.json'; // eslint-disable-line import/no-unresolved
+import { IWDP } from './iwdp';
```

Comment:
actually, we should probably have ios-driver export IWDP so that it can be used by xcuitest-driver, unless it simply calls ios-driver's `startRealDevice` method and `stop` method without modifications (which i don't think is the case)